src/main/java/com/crud/tasks/service/DbService.java

```diff
@@ -14,7 +14,10 @@
 
     public List<Task> getAllTasks() {
         return repository.findAll();
+    }
 
+    public Task getTask(Long id) {
+        return repository.findById(id).orElse(null);
```

Comment:
to nie jest dobre rozwiazanie. Lepiej bylo by napisac np .orElseThrow(()->TaskNotFoundException (rozszezajacy RuntimeException)
Mozesz tez dorzucic ResponseStatus
https://www.baeldung.com/spring-response-status